tests/x11/remote_desktop/windows_network_setup.pm

```diff
@@ -32,6 +32,9 @@ sub run {
     $self->run_in_powershell(cmd => 'Get-NetIPAddress -InterfaceAlias $a.name', tags => 'win-remote-desktop');
     $self->run_in_powershell(cmd => 'New-NetIPAddress -InterfaceAlias $a.name -IPAddress 10.0.2.18 -PrefixLength 24 -DefaultGateway 10.0.2.2 -Confirm:$false', tags => 'win-remote-desktop');
     approve_network_popup;
+    # Imcomplete input on powershell because executing command is too fast
+    # Poo#109091
```

Comment:
Suggest change:
We may miss some characters due to type command too fast after network popup, so wait several seconds as a workaround,
See poo#109091

Reply:
updated.